scene/show-labels-on-layer-in-3d/src/main/java/com/esri/samples/show_labels_on_layer_in_3d/ShowLabelsOnLayerIn3dSample.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2021 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package com.esri.samples.show_labels_on_layer_in_3d;
+
+import com.esri.arcgisruntime.arcgisservices.LabelingPlacement;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.labeling.ArcadeLabelExpression;
+import com.esri.arcgisruntime.symbology.ColorUtil;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.StackPane;
+import javafx.scene.paint.Color;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.arcgisservices.LabelDefinition;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+import com.esri.arcgisruntime.symbology.TextSymbol;
+
+public class ShowLabelsOnLayerIn3dSample extends Application {
+
+  private SceneView sceneView;
+
+  @Override
+  public void start(Stage stage) {
+
+    try {
+
+      // create stack pane and JavaFX app scene
+      StackPane stackPane = new StackPane();
+      Scene fxScene = new Scene(stackPane);
+
+      // set title, size, and add the JavaFX scene to stage
+      stage.setTitle("Show Labels on Layer in 3D");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(fxScene);
+      stage.show();
+
+      // create a scene from an ArcGISOnline portal item
+      var portalItem = new PortalItem(
```

Comment:
I'm onboard with the uses of `var`  in the Sample. Think they make sense and it remains very clear what Class is being used.